test/k8sT/fqdn.go

```diff
@@ -30,22 +30,37 @@ var _ = Describe("K8sFQDNTest", func() {
 		backgroundCancel context.CancelFunc = func() { return }
 		backgroundError  error
 
-		bindManifest = ""
-		demoManifest = ""
+		bindManifest  = ""
+		demoManifest  = ""
+		httpdManifest = ""
 
 		apps    = []string{helpers.App2, helpers.App3}
 		appPods map[string]string
 
 		worldTarget          = "http://world1.cilium.test"
-		worldTargetIP        = "192.168.9.10"
+		worldTargetIP        = "192.168.9.10" //set in BeforeAll if on eks
 		worldInvalidTarget   = "http://world2.cilium.test"
-		worldInvalidTargetIP = "192.168.9.11"
+		worldInvalidTargetIP = "192.168.9.11" //set in BeforeAll if on eks
 	)
 
 	BeforeAll(func() {
 		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
-		bindManifest = helpers.ManifestGet(kubectl.BasePath(), "bind_deployment.yaml")
-		demoManifest = helpers.ManifestGet(kubectl.BasePath(), "demo.yaml")
+
+		if helpers.IsIntegration(helpers.CIIntegrationEKS) {
+			ip1, err := kubectl.GetNodeIPByLabel("k8s1")
+			Expect(err).Should(BeNil(), "Unable to get k8s1 node ip")
+			ip2, err := kubectl.GetNodeIPByLabel("k8s2")
+			Expect(err).Should(BeNil(), "Unable to get k8s2 node ip")
+			worldTargetIP = ip1
+			worldInvalidTargetIP = ip2
+
+			By("Applying httpd host network pods")
+			httpdManifest = helpers.ManifestGet(kubectl.BasePath(), "fqdn-httpd.yaml")
+
+			res := kubectl.ApplyDefault(httpdManifest)
+			res.ExpectSuccess("httpd server cannot be deployed")
+		}
+
```

Comment:
where's the else statement? I'm asking because previously we were doing `bindManifest = helpers.ManifestGet(kubectl.BasePath(), "bind_deployment.yaml")` and `demoManifest = helpers.ManifestGet(kubectl.BasePath(), "demo.yaml")` but we aren't doing it anymore. I see the else statement below for the `demo.yaml` but not for the bindManifest`

Reply:
In this specific case, it's in the CI provisioning scripts :/ This just replicates a server runnning on docker. There are other places where we have an if like this. There, the else is just the default.